doc/en/api/1.0.0/wmslayers.yaml

```diff
@@ -139,55 +139,109 @@ paths:
           schema:
             $ref: "#/definitions/WMSStoreLayerInfo"
           examples:
-            application/xml: |
+            application/xml: |              
               <wmsLayer>
-                <name>dem</name>
-                <nativeName>usgs:dem</nativeName>
-                <namespace>
-                  <name>cite</name>
-                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geoserver/rest/namespaces/cite.xml" type="application/xml"/>
-                </namespace>
-                <title>Digital elevation model</title>
-                <description>USGS GTOPO30 digital elevation model</description>
-                <abstract>USGS GTOPO30 digital elevation model</abstract>
-                <keywords>
-                  <string>WCS</string>
-                  <string>GeoTIFF</string>
-                  <string>W100N40</string>
-                </keywords>
-                <nativeCRS>GEOGCS[&quot;WGS 84&quot;, &#xd;
-                DATUM[&quot;World Geodetic System 1984&quot;, &#xd;
-                  SPHEROID[&quot;WGS 84&quot;, 6378137.0, 298.257223563, AUTHORITY[&quot;EPSG&quot;,&quot;7030&quot;]], &#xd;
-                  AUTHORITY[&quot;EPSG&quot;,&quot;6326&quot;]], &#xd;
-                PRIMEM[&quot;Greenwich&quot;, 0.0, AUTHORITY[&quot;EPSG&quot;,&quot;8901&quot;]], &#xd;
-                UNIT[&quot;degree&quot;, 0.017453292519943295], &#xd;
-                AXIS[&quot;Geodetic longitude&quot;, EAST], &#xd;
-                AXIS[&quot;Geodetic latitude&quot;, NORTH], &#xd;
-                AUTHORITY[&quot;EPSG&quot;,&quot;4326&quot;]]</nativeCRS>
-                <srs>EPSG:4326</srs>
-                <nativeBoundingBox>
-                  <minx>-100.0</minx>
-                  <maxx>-60.000000000015994</maxx>
-                  <miny>-9.999999999980012</miny>
-                  <maxy>39.99999999999999</maxy>
-                  <crs>EPSG:4326</crs>
-                </nativeBoundingBox>
-                <latLonBoundingBox>
-                  <minx>-100.0</minx>
-                  <maxx>-60.000000000015994</maxx>
-                  <miny>-9.999999999980012</miny>
-                  <maxy>39.99999999999999</maxy>
-                  <crs>EPSG:4326</crs>
-                </latLonBoundingBox>
-                <projectionPolicy>FORCE_DECLARED</projectionPolicy>
-                <enabled>true</enabled>
-                <metadata>
-                  <entry key="cachingEnabled">false</entry>
-                </metadata>
-                <store class="wmsStore">
-                  <name>cite:altgs</name>
-                  <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://localhost:8080/geoserver/rest/workspaces/cite/wmsstores/altgs.xml" type="application/xml"/>
-                </store>
+                 <name>dem</name>
```

Comment:
Some small difference of indenting here, just making a note that it masks the small change you performed in the example.

Reply:
will fix